packages/gatsby/src/redux/actions.js

```diff
@@ -403,6 +403,29 @@ actions.deleteNode = (options: any, plugin: Plugin, args: any) => {
   // Always get node from the store, as the node we get as an arg
   // might already have been deleted.
   const node = getNode(id)
+  if (plugin) {
+    const pluginName = plugin.name
+
+    if (node && typeOwners[node.internal.type] !== pluginName)
+      throw new Error(stripIndent`
+          The plugin "${pluginName}" deleted a node of a type owned by another plugin.
+
+          The node type "${node.internal.type}" is owned by "${
+        typeOwners[node.internal.type]
+      }".
+
+          If you copy and pasted code from elsewhere, you'll need to pick a new type name
+          for your new node(s).
```

Comment:
I know this was copied from `createNode`, but (to me) this note about picking different type name doesn't really work for deleting nodes. Maybe let's just remove lines 417-418

Reply:
Yes seems logic, sorry about that 😊